### DISCUSSION TITLE
A Practice Diff Feature in the Editor Aimed at Allowing Spectators

### DISCUSSION THREAD

**Discussion by mojobojo** [ppy/osu#20778] [Ideas] [answered]
Basically what I am proposing is a way to create a "hyperlink" to a section in a map. Essentially making creating and using a practice diff easier and allowing spectators to still download the map you are playing so they can still watch.

**Comment by peppy** [comment, marked as answer]
We are intending on adding a mod flow (or similar) for this. It will not involve the editor.
